app/Category.php

```diff
@@ -10,11 +10,25 @@ class Category extends Model
     
     public $timestamps = false;
 
+    public function index($number)
+    {
+        $categories = $this->orderBy('id', 'DESC')->paginate($number);
+        return $categories;
```

Comment:
it's ok no need to put it in variable, it's ok like this `return $this->orderBy('id', 'DESC')->paginate($number);`